Header header
string name
string src
float32[] locNED
float32[] J
float32[] shape
float32[] uvwLoc
float32[] hriTouchXs
float32[] hriTouchYs
float32[] hriTouchZs